src/helm.ts

```diff
@@ -25,8 +25,11 @@ export const initializeHelm = (
 export const installUrbanOS = (
   classRef: TerraformStack,
   dependsOn: DependsOn
-) =>
-  new Release(classRef, "UrbanOSHelmRelease", {
+) => {
+  // Waits 5 minutes because Postgres marks as complete before it's ready
+  Atomics.wait(new Int32Array(new SharedArrayBuffer(4)), 0, 0, 3000);
+
+  return new Release(classRef, "UrbanOSHelmRelease", {
```

Comment:
I don't believe this will work

Anything in a `.ts` file only generates the tf.out file. Then the tf.out file is applied by terraform.

My understanding is this will just add a 5 minute delay in the tf.out file being generated, with no affect on the actual `tf apply` that creates resources from the tf.out.

Other ideas would be, maybe create a helm chart that's entire job is just to take a while to install? That way you could set up a dependency that creates a 5 minute delay incidentally.

```javascript
delay = install5MinDelay([dependsOn: postgres])
installUrbanOS[(dependsOn: [..., delay])

// by depending on the delay, urbanos install is delayed
// this is a pretty gross idea tho
```
The "proper" way would be to make the services that depend on postgres, able to attempt reconnecting themselves, but I understand how that would be a larger set of work.